doc/content/integrations/mqtt/Connecting to Node-RED/_index.md

```diff
@@ -15,50 +17,69 @@ First, you need to run the Node-RED by using the following command in terminal:
 
 {{< figure src="nodered_dashboard.png" alt="Node-RED dashboard" >}}
 
 On the left side, you can see various types of nodes (they can also be found in the [Node-RED library](https://flows.nodered.org/)) that can be used in order to build flows, although in this guide only a few simple inbuilt nodes are used. 
 
-The Console provides the information needed for completing this integration, under **MQTT** submenu from **Integrations** menu within desired application.
+The in the {{% tts %}} Console provides the information needed for completing this integration, under **MQTT** submenu from **Integrations** menu within desired application.
 
 {{< figure src="console_info.png" alt="MQTT Server connection information" >}}
 
-In this example, since TTS is running locally, the inbuilt MQTT Server is deployed on port 1883 for insecure connection and on port 8883 for the TLS-secured one.
+In this example, since {{% tts %}} is running locally, the inbuilt MQTT Server is deployed on port 1883 for insecure connection and on port 8883 for the TLS-secured one.
 
-## Receive events and messages
+## Receive Events and Messages
 
 This section follows the process of setting up a flow which subscribes and listens to the events and messages that are being published by the MQTT Server. 
 
-First, find the `mqtt in` node in the library section and place it on the flow dashboard. This node connects to the MQTT broker and subscribes to messages from the specified topic. Double-click on the node to configure its properties.
+First, find the **mqtt in** node in the library section and place it on the flow dashboard. This node connects to the MQTT broker and subscribes to messages from the specified topic. Double-click on the node to configure its properties.
 
-{{< figure src="node_properties.png" alt="mqtt in node properties" >}}
+{{< figure src="mqtt_in_node_properties.png" alt="mqtt in node properties" >}}
 
 In the **Server** dropdown menu, select **Add new mqtt-broker** and click on the button besides to edit it. 
 
-{{< figure src="node_connection.png" alt="COnfiguring MQTT Server connection information" >}}
+{{< figure src="mqtt_in_node_connection.png" alt="COnfiguring MQTT Server connection information" >}}
 
-In the **Connection** tab, under **Server**, you should provide MQTT server's URL - if it runs locally, then type **localhost**. The **Port** value depends on whether the connection is to be set up as a secure one or not. In this example, we are using TLS, so the **Port** value is set to 8883. In this case you should also check the **Enable secure (SSL/TLS) connection** box.
+In the **Connection** tab, under **Server**, you should provide MQTT Server's URL - if it runs locally, then type **localhost**. The **Port** value depends on whether the connection is to be set up as a secure one or not. In this example, TLS is being used, so the **Port** value is set to 8883. In this case you should also check the **Enable secure (SSL/TLS) connection** box.
```

Comment:
>MQTT Server's URL - if it runs locally, then type **localhost**.

- It's not a URL; it's an address
- They can find the address in MQTT under Integrations. That is the only place we should refer to; do not mention "if it runs locally" in this guide; if the server runs locally, the address shown in the Console is already `localhost`

<img width="1147" alt="Screenshot 2020-05-29 at 09 10 32" src="https://user-images.githubusercontent.com/13334001/83231774-5910fb80-a18c-11ea-8d89-17aca4276682.png">